ENIGMAsystem/SHELL/Universal_System/fileio.h

```diff
@@ -33,7 +33,7 @@ void    file_text_writeln(int fileid);
 std::string file_text_read_string(int fileid);
 std::string file_text_read_all(int fileid);
 double  file_text_read_real(int fileid);
-void    file_text_readln(int fileid);
+std::string file_text_readln(int fileid);
```

Comment:
These no longer line up.

Reply:
Well, it's more aligned now with the other read functions, I could format the whole header, but I avoided shitting up the diff for that unlike #1141.

Reply:
I can realign it with a smaller diff you allow `using std::string;` actually. Would you like that?

Reply:
Ok: 05684a59da99acc9ac6f5d4e10004f4c1d573e70

I realigned the header by using string.